src/app.js

```diff
@@ -27,3 +27,7 @@ ReactDOM.render(
   </Provider>,
   document.getElementById('app')
 );
+
+window.Intercom("boot", {
```

Comment:
not sure if this is the best place to put this. if the `<script>` tag from `index.html` is moved to the bottom of the `<body>` tag, which it should be, then we might run into a race-condition because `window.Intercom` might be undefined.